cmd/contractcmd/deploy_erc20.go

```diff
@@ -46,7 +46,8 @@ func newDeployERC20Cmd() *cobra.Command {
 	}
 	networkoptions.AddNetworkFlagsToCmd(cmd, &deployERC20Flags.Network, true, deployERC20SupportedNetworkOptions)
 	deployERC20Flags.PrivateKeyFlags.AddToCmd(cmd, "as contract deployer")
-	deployERC20Flags.chainFlags.AddToCmd(cmd, "deploy the ERC20 contract", true)
+	deployERC20Flags.chainFlags.SetEnabled(true, true, false, false, true)
```

Comment:
nit: it's hard to read true true false false true to get what it means. pls add comment describing it
pls feel free to ignore this comment